adagrams/game.py

```diff
@@ -1,11 +1,118 @@
+import random
+import string
 def draw_letters():
-    pass
+    hand_of_letters = []
+    LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+    }
+    
+    #iterate through the dict to convert letter_pool to a list
+    LETTER_POOL_list = []  
+    for letter,count in LETTER_POOL.items():
+        while count > 0:
+            LETTER_POOL_list.append(letter)
+            count -=1    
```

Comment:
Prefer not to use `while` loop here since we know how many times we need to loop. A for range loop would work better: `for _ in range(count):`

A Pythonic way of writing this would be to use list comprehension. Lines 35-39 can be replaced with:

```python
letter_pool_list = [letter for letter, count in LETTER_POOL.items() for _ in range(count)]
```

Reply:
This is exactly what I was talking about being Pythonic. Thank you! I will try to absorb it.